src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -101,13 +127,13 @@ class PAITrainingService implements TrainingService {
             throw new Error('PAI Cluster config is not initialized');
         }
 
-        const paiTrialJob: PAITrialJobDetail | undefined = this.trialJobsMap.get(trialJobId);
+        const paiBaseTrialJob: PAITrialJobDetail | undefined = this.trialJobsMap.get(trialJobId);
 
-        if (paiTrialJob === undefined) {
+        if (paiBaseTrialJob === undefined) {
```

Comment:
throw new Error to keep style consistent

Reply:
fixed.